--- 10_linux	2020-04-14 18:56:00.112887371 +0200
+++ 10_linux	2020-04-14 18:56:54.340924243 +0200
@@ -169,12 +169,12 @@
   if [ -d /sys/firmware/efi ] && [ "x${GRUB_USE_LINUXEFI}" = "xtrue" ]; then
     sed "s/^/$submenu_indentation/" << EOF
 	echo	'$(echo "$message" | grub_quote)'
-	linuxefi ${rel_dirname}/${basename} ${root_device} ${args}
+	\$linux ${rel_dirname}/${basename} ${root_device} ${args}
 EOF
   else
     sed "s/^/$submenu_indentation/" << EOF
 	echo	'$(echo "$message" | grub_quote)'
-	linux	${rel_dirname}/${basename} ${root_device} ${args}
+	\$linux	${rel_dirname}/${basename} ${root_device} ${args}
 EOF
   fi
   if test -n "${initrd}" ; then
@@ -183,12 +183,12 @@
     if [ -d /sys/firmware/efi ] && [ "x${GRUB_USE_LINUXEFI}" = "xtrue" ]; then
       sed "s/^/$submenu_indentation/" << EOF
 	echo	'$(echo "$message" | grub_quote)'
-	initrdefi ${rel_dirname}/${initrd}
+	\$initrd ${rel_dirname}/${initrd}
 EOF
     else
       sed "s/^/$submenu_indentation/" << EOF
 	echo	'$(echo "$message" | grub_quote)'
-	initrd	${rel_dirname}/${initrd}
+	\$initrd	${rel_dirname}/${initrd}
 EOF
     fi
   fi
